cmd/influx/v1_dbrp.go

```diff
@@ -0,0 +1,386 @@
+	return nil
+}
+
+var v1DBRPDeleteFlags struct {
+	ID  string       // Specifies the mapping ID to filter on
+	Org organization // required  // Specifies the organization ID to filter on
+}
+
+func v1DBRPDeleteCmd(f *globalFlags, opt genericCLIOpts) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delete",
+		Short: "Delete DBRP",
+		RunE:  checkSetupRunEMiddleware(&flags)(v1DBRPDeleteF),
+	}
+
+	f.registerFlags(opt.viper, cmd)
+	registerPrintOptions(opt.viper, cmd, &v1DBRPCRUDFlags.hideHeaders, &v1DBRPCRUDFlags.json)
+	v1DBRPCreateFlags.Org.register(opt.viper, cmd, false)
+	cmd.Flags().StringVarP(&v1DBRPDeleteFlags.ID, "id", "i", "", "The dbrp ID (required)")
+	_ = cmd.MarkFlagRequired("id")
+	return cmd
+}
+
+func v1DBRPDeleteF(cmd *cobra.Command, args []string) error {
+	if err := v1DBRPCreateFlags.Org.validOrgFlags(&flags); err != nil {
+		return err
+	}
+	orgSvc, err := newOrganizationService()
+	if err != nil {
+		return err
+	}
+
+	orgID, err := v1DBRPDeleteFlags.Org.getID(orgSvc)
+	if err != nil {
+		return err
+	}
+
+	s, err := newV1DBRPService()
+	if err != nil {
+		return err
+	}
+
+	id, err := influxdb.IDFromString(v1DBRPDeleteFlags.ID)
+	if err != nil {
+		return err
+	}
+
+	dbrpToDelete, err := s.FindByID(context.Background(), orgID, *id)
+	if err != nil {
+		return err
+	}
+
+	if err := s.Delete(context.Background(), orgID, *id); err != nil {
+		return err
+	}
+	return v1WriteDBRPs(cmd.OutOrStdout(), v1DBRPPrintOpt{
+		jsonOut:     v1DBRPCRUDFlags.json,
+		hideHeaders: v1DBRPCRUDFlags.hideHeaders,
+		mapping:     dbrpToDelete,
+	})
+}
+
+var v1DBRPUpdateFlags struct {
+	ID      string       // Specifies the mapping ID to filter on
+	Org     organization // required  // Specifies the organization ID to filter on
+	DB      string       //InfluxDB v1 database
+	Default *bool
+	RP      string // InfluxDB v1 retention policy
+}
+
+func v1DBRPUpdateCmd(f *globalFlags, opt genericCLIOpts) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update DBRP",
+		RunE:  checkSetupRunEMiddleware(&flags)(v1DBRPUpdateF),
+	}
+	f.registerFlags(opt.viper, cmd)
+	registerPrintOptions(opt.viper, cmd, &v1DBRPCRUDFlags.hideHeaders, &v1DBRPCRUDFlags.json)
+	v1DBRPCreateFlags.Org.register(opt.viper, cmd, false)
+	cmd.Flags().StringVarP(&v1DBRPUpdateFlags.ID, "id", "i", "", "The dbrp ID (required)")
+	_ = cmd.MarkFlagRequired("id")
+	cmd.Flags().StringVarP(&v1DBRPUpdateFlags.RP, "rp", "r", "", "The dbrp InfluxDB v1 retention policy")
+	cmd.Flags().Bool("default", false, "If dbrp InfluxDB v1 retention policy is default, (only changes when explicitly set)") // note for update we only care about update flags that the user set
+	return cmd
+}
+
+func v1DBRPUpdateF(cmd *cobra.Command, args []string) error {
+	if err := v1DBRPCreateFlags.Org.validOrgFlags(&flags); err != nil {
+		return err
+	}
+	if defaultFlg := cmd.Flags().Lookup("default"); defaultFlg.Changed {
+		defaultBool, _ := cmd.Flags().GetBool("default")
+		defaultBool = false
```

Comment:
Should this be here? Won't `v1DBRPUpdateFlags.Default` always end up `&false`?

Reply:
Good eye, I was using this to debug and forgot to remove it.  THANKS.